index.html

```diff
@@ -1,75 +1,245 @@
+    <link href="https://cdn.jsdelivr.net/fontawesome/4.4.0/css/font-awesome.min.css"
+          rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,300,200,600'
+          rel='stylesheet' type='text/css'>
     <style>
-     body {
-     	font-family: 'Source Sans Pro', sans-serif;
-     	font-weight: 300
-     }
+        body {
+            font-family: 'Source Sans Pro', sans-serif;
+            font-weight: 300;
+        }
+
+        .jumbotron {
+            padding-top: 1em;
+        }
+
+        #builds-table {
+            display: none;
+        }
+
+        #loading-icon {
+            display: block;
+            text-align: center;
+            margin-bottom: 2em;
+        }
+
+        footer {
+            text-align: center;
+        }
     </style>
-  </head>
-  <body>
+</head>
+
+<body>
 <nav class="navbar navbar-dark bg-inverse navbar-fixed-top">
-<div class="container">
-  <a class="navbar-brand" href="#">Glowstone++</a>
-  <ul class="nav navbar-nav pull-right">
-    <li class="nav-item">
-      <a class="nav-link" href="https://gitter.im/GlowstonePlusPlus/GlowstonePlusPlus">Chat</a>
-    </li>
-    <li class="nav-item">
-      <a class="nav-link" href="https://github.com/GlowstonePlusPlus/GlowstonePlusPlus/wiki">Wiki</a>
-    </li>
-  </ul>
-</div>
+    <div class="container">
+        <a class="navbar-brand" href="#">Glowstone++</a>
+        <ul class="nav navbar-nav pull-right">
+            <li class="nav-item">
+                <a class="nav-link"
+                   href="http://webchat.esper.net/?nick=webchat-.....&channels=glowstone++&prompt=0">IRC</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link"
+                   href="https://gitter.im/GlowstonePlusPlus/GlowstonePlusPlus">Gitter</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link"
+                   href="https://github.com/GlowstonePlusPlus/GlowstonePlusPlus/wiki">Wiki</a>
+            </li>
+        </ul>
+    </div>
 </nav>
-    <header class="jumbotron jumbotron-fluid" style="background: #FF6F00;color: #fff;height:100%;margin-top:54px">
+
+<header class="jumbotron jumbotron-fluid"
+        style="background: #FF6F00;color: #fff;height:100%;margin-top:54px">
+    <div class="container">
+        <h1>Minecraft's future is bright.</h1>
+        <p class="lead"><b>Meet Glowstone++.</b> It's open, fast, and
+            compatible.</p>
+        <p>Glowstone++ is a completely open-source Minecraft server that
+            doesn't rely on any of Mojang's code - meaning we're free to
+            develop for
+            your needs, and you're free to change it however you want.</p>
+        <a href="https://bamboo.gserv.me/browse/GSPP-SRV/latestSuccessful/artifact/shared/Server-JAR/glowstone%2B%2B-1.8.9-SNAPSHOT.jar"
+           class="btn btn-primary"><i class="fa fa-download"></i> Download for
+            1.8.9 (Experimental)</a>
```

Comment:
![](https://dl.dropboxusercontent.com/u/1547120/ShareX/2016/February/firefox_2016-02-21_19-04-54.png)

I prefer it with a capital E, though it'd be better if it was smaller than that if we were going that route.